packages/client/src/client/Web/cacheUpdateModal.tsx

```diff
@@ -36,19 +18,24 @@ export const showCacheUpdateModal = () => {
     <ThemeProvider>
```

Comment:
aside: considering refactoring this to use [`<dialog>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dialog) and include in router, rather than this createRoot gubbins
(as a side effect, we should not have popups appearing in _every_ window)